### PR TITLE
Make search line termination a circle, like 2D UI

### PR DESCRIPTION
**Pull request by jherico**
## Testing

Use the handControllerGrab.js script included in this PR to search for and grab items.  The search laser should appear with a circle in place of the former sphere termination point.  All other behavior should be the same.

### CODE REVIEW

**Comment by jherico** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;W0wWwrpP&#x2F;75-make-the-look-of-the-2d-overlay-lasers-and-termination-point-consistent-with-the-3d-lasers-and-termination-point"> Make the look of the 2D overlay lasers and termination point consistent with the 3D lasers and termination point</a>

File: scripts/system/controllers/handControllerGrab.js [hunk 1]
```diff
@@ -547,23 +555,34 @@ function MyController(hand) {
 
     var SEARCH_SPHERE_ALPHA = 0.5;
     this.searchSphereOn = function (location, size, color) {
+        
+        var rotation = Quat.lookAt(location, Camera.getPosition(), Vec3.UP);
+        var brightColor = colorPow(color, 0.1);
+        print("bright color " + brightColor.red + " " + brightColor.green + " " + brightColor.blue);
```
Comment (ZappoMan) [inline scripts/system/controllers/handControllerGrab.js]: do you still want this debugging here?

Comment (jherico) [inline scripts/system/controllers/handControllerGrab.js]: thanks, fixed.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0d811c489aa3464f9a75761581078ad5c0a50b29 available at: https://gist.github.com/ab56018798c9a5f50a53108672ae1763<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8255-0d811c4.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8255-0d811c4.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: c462b8c387559f14150d4d9416eab2d8c3d89e15 available at: https://gist.github.com/ab56018798c9a5f50a53108672ae1763<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8255-c462b8c.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8255-c462b8c.dmg

**Comment by RyanDowne** [comment]
This is looking cool. I think I want Philip to take a look first. What I am seeing is that depending on how close you are to the termination circle, you can see the laser continue through it so it doesn't always look 'right'. I am guessing that is a non trivial problem to solve.

**Comment by hifi-gustavo** [comment]
Build console output for commit: b31300406a63fd39c34c570cc554f4ceadc87161 available at: https://gist.github.com/ab56018798c9a5f50a53108672ae1763<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8255-b313004.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8255-b313004.dmg

**Comment by PhilipRosedale** [comment]
Tested for many different objects and ranges.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1145c3b59006e2bf3378f80a8d85e5f48b9858dc available at: https://gist.github.com/ab56018798c9a5f50a53108672ae1763<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8255-1145c3b.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8255-1145c3b.dmg

**Pull request merged**